index.js

```diff
@@ -1,59 +1,84 @@
-'use strict';
+/**
+ * isLink Function
+ * Checks if the object has sys.type "Link"
+ * @param object
+ */
+const isLink = (object) => object && object.sys && object.sys.type === 'Link';
+/**
+ * findNormalizableArray
+ *
+ * @param array
+ * @param predicate
+ * @return {*}
+ */
+const findNormalizableArray = (array, predicate) => {
```

Comment:
I find this name very confusing, because it tries to find a normalizable link in an array, and not an array itself.

Reply:
well.. true. renaming this.